### DISCUSSION TITLE
Make the path argument to cli schema snapshot optional

### DISCUSSION THREAD

**Discussion by claytongulick** [directus/directus#14073] [Feature Requests]
If the path argument is missing, the schema should just be written to stdout.

This enables piping and auto schema sync scripts via ssh, etc...

PR incoming.

**Comment by claytongulick** [comment]
https://github.com/directus/directus/pull/14074

**Comment by rijkvanzanten** [comment]
Implemented in v9.13 through https://github.com/directus/directus/pull/14074 🙂 Thanks @claytongulick! ❤️
